monkey/agent_plugins/exploiters/snmp/manifest.yaml

```diff
@@ -0,0 +1,21 @@
+name: SNMP
+plugin_type: Exploiter
+supported_operating_systems:
+  - linux
+  - windows
+target_operating_systems:
+  - linux
+title: SNMP Exploiter
+version: 1.0.0
+description: Attempts remote command execution over SNMP using known credentials.
+safe: true
+remediation_suggestion: >-
+    Configure SNMP to use read-only communities.
+
+    Limit access over SNMP to trusted hosts.
+
+    Use SNMPv3 with the authPriv security level if possible.
+
+
+    The machine is vulnerable to an SNMP attack.
+    An Infection Monkey Agent executed a command over SNMP using stolen/configured credentials.
```

Comment:
Should we be more specific in here, call out the CVE, and recommend they Install a patched version?

Reply:
+1
We can do it the way we've done it for Zerologon.
We should also do this for Hadoop.

Reply:
Is CVE-2020-15862 the correct one?

Reply:
Looks good.

Reply:
> We should also do this for Hadoop.

@shreyamalviya  I don't believe there's any specific CVE associated with Hadoop.